README.md

```diff
@@ -48,27 +48,27 @@ default is 0755
-| DVC_LB_PROXY_PLATFORMDATA_PLATFORMVERSION              | String        |         |          |                                                                                 |
-| DVC_LB_PROXY_PLATFORMDATA_DEVICEMODEL                  | String        |         |          |                                                                                 |
-| DVC_LB_PROXY_PLATFORMDATA_PLATFORM                     | String        |         |          |                                                                                 |
-| DVC_LB_PROXY_PLATFORMDATA_HOSTNAME                     | String        |         |          |                                                                                 |
-| DVC_LB_PROXY_SDKCONFIG_EVENT_FLUSH_INTERVAL_MS         | Duration      |         |          | The interval at which events are flushed to the events api in milliseconds.     |
-| DVC_LB_PROXY_SDKCONFIG_CONFIG_POLLING_INTERVAL_MS      | Duration      |         |          | The interval at which the SDK polls the config CDN for updates in milliseconds. |
-| DVC_LB_PROXY_SDKCONFIG_REQUEST_TIMEOUT                 | Duration      |         |          | The timeout for requests to the config CDN and events API in milliseconds.      |
-| DVC_LB_PROXY_SDKCONFIG_DISABLE_AUTOMATIC_EVENT_LOGGING | True or False | false   |          | Whether to disable automatic event logging. Defaults to false.                  |
-| DVC_LB_PROXY_SDKCONFIG_DISABLE_CUSTOM_EVENT_LOGGING    | True or False | false   |          | Whether to disable custom event logging. Defaults to false.                     |
-| DVC_LB_PROXY_SDKCONFIG_MAX_EVENT_QUEUE_SIZE            | Integer       |         |          | The maximum number of events to be in the queue before dropping events.         |
-| DVC_LB_PROXY_SDKCONFIG_FLUSH_EVENT_QUEUE_SIZE          | Integer       |         |          | The minimum number of events to be in the queue before flushing events.         |
-| DVC_LB_PROXY_SDKCONFIG_CONFIG_CDN_URI                  | String        |         |          | The URI of the Config CDN - leave unspecified if not needing an outbound proxy. |
-| DVC_LB_PROXY_SDKCONFIG_EVENTSAPIURI                    | String        |         |          | The URI of the Events API - leave unspecified if not needing an outbound proxy. |
\ No newline at end of file
+| KEY                                                      | TYPE          | DEFAULT | REQUIRED | DESCRIPTION                                                                     |
+|----------------------------------------------------------|---------------|---------|----------|---------------------------------------------------------------------------------|
+| DEVCYCLE_PROXY_CONFIG                                    | String        |         |          | The path to a JSON configuration file.                                          |
```

Comment:
Will we need to do a major version increase for this?

Reply:
Yep